anchor/anchorRecord.go

```diff
@@ -17,22 +17,32 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
-//AnchorRecord is used to construct anchor chain
+// CurrentAnchorVersion is the current anchor version
+const CurrentAnchorVersion int = 1
+
+// AnchorRecord is used to construct the anchor chain. The Factom Protocol writes an anchor into
+// a parent blockchain (Bitcoin or Ethereum) approximately every 10 minutes. The accumulated entries
+// from the previous 10 minutes are organized into a Directory Block (DBlock). The information
+// from the directory block is stored in the AnchorRecord.
 type AnchorRecord struct {
-	AnchorRecordVer int
-	DBHeight        uint32 `json:",omitempty"` // The only directory block height included in this anchor
-	KeyMR           string `json:",omitempty"` // Merkle root of the only directory block included in this anchor
+	AnchorRecordVer int    // the only supported version seems to be 1
```

Comment:
as noted in another comment, version 2 exists as well

Reply:
Fixed this comment